bluemira/utilities/opt_variables.py

```diff
@@ -27,9 +28,9 @@ class OptVarVarDictValueT(TypedDict, total=False):
     """Typed dictionary for a the values of an OptVariable from a var_dict."""
 
     value: float
-    lower_bound: float
-    upper_bound: float
-    fixed: bool
+    lower_bound: NotRequired[float]
+    upper_bound: NotRequired[float]
+    fixed: NotRequired[bool]
 
```

Comment:
These are fine beucase this type is used in the `adjust_variables` function, i.e. lower, upper etc can be none for that, no adjustment